Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml

```diff
@@ -0,0 +1,35 @@
+- type: job
+  id: Paramedic
+  name: job-name-paramedic
+  description: job-description-paramedic
+  playTimeTracker: JobParamedic
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Medical
+      time: 9000
+    - !type:OverallPlaytimeRequirement
+      time: 54000
+  startingGear: ParamedicGear
+  icon: "Paramedic"
+  supervisors: job-supervisors-cmo
+  access:
+  - Medical
+  - Maintenance
+  - External
+  - Paramedic
+  extendedAccess:
+  - Chemistry
+
+- type: startingGear
+  id: ParamedicGear
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitParamedic
+    back: ClothingBackpackMedicalFilled
+    shoes: ClothingShoesColorWhite
+    id: ParamedicPDA
+    ears: ClothingHeadsetMedical
+    pocket1: HandheldCrewMonitor
```

Comment:
This is a lategame tech, please don't add it to them.

Reply:
Should we remove it from laying around on most stations and from the CMO too?  Serious question, not sarcasm.

Reply:
Removed.